api/v1/Makefile

```diff
@@ -51,4 +51,4 @@ clean:
 
 .PHONY: deps
 deps:
-	@$(PROTOC) --version | grep 3.11.4 || (echo "Error: protoc 3.11.4 is required" && exit 1)
+	@$(PROTOC) --version | grep 3.12.3 || (echo "Error: protoc 3.12.3 is required" && exit 1)
```

Comment:
This change does not seem relevant to the PR. I'm always in favor of updating things but this PR is also marked to be backported to 1.8. Ideally we should avoid YOLOing backporting vendor / generated changes into older branches.

Reply:
I agree with you in general but for this particular case I think it is fine as `protoc` is only used for to generate files. When using the newer version, the generated files are strictly identical.

Reply:
I removed this commit and opened #12677.